.circleci/config.yml

```diff
@@ -58,14 +58,12 @@ jobs:
       - run:
           name: Build application Docker image
           command: |
-            docker build -t hmpps-integration-api .
+            make build
```

Comment:
We can also replace the `./gradlew test` within the `unit-tests` job with `make unit-tests`. I guess we could also have one for linting?